docker/pudl_etl.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/bash
+
+pudl_setup \
+    --pudl_in $CONTAINER_PUDL_IN \
+    --pudl_out $CONTAINER_PUDL_OUT
+ferc1_to_sqlite \
+    --clobber \
+    --gcs-cache-path gs://zenodo-cache.catalyst.coop \
```

Comment:
Should we refer to the cache path using the `$GCS_CACHE` environment variable here and below?